src/BuildScriptGeneratorCli/readme.md

```diff
@@ -0,0 +1,11 @@
+# Oryx CLI Structure
+### 1. Command Files
+Option and argument inputs are handled through binders. The command file only define the subcommands and register options and arguments, and export them to be added to root command in `Program.cs`.
```

Comment:
nit: comment still applies about capping `.md` file lines to 120 characters where we can for readability